app/models/peep.rb

```diff
@@ -0,0 +1,13 @@
+class Peep
+  include DataMapper::Resource
+
+  property :id,   Serial
+  property :when, DateTime, :default => lambda{ |p,s| DateTime.now}
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `p`. If it's necessary, use `_` or `_p` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Lint/UnusedBlockArgument: Unused block argument - `s`. If it's necessary, use `_` or `_s` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.<br>Style/Lambda: Use the new lambda literal syntax `->(params) {...}`.